documentation/source/quickstart.rst

```diff
@@ -18,104 +18,92 @@ Cocotb has the following requirements:
-`pip User Guide <https://pip.pypa.io/en/stable/user_guide/#user-installs/>`_.
+    conda install -c msys2 m2-base m2-make m2w64-toolchain libpython
 
-To install the development version of cocotb:
+Native Installation for Debian/Ubuntu-based Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: bash
 
-    git clone https://github.com/cocotb/cocotb
-    pip3 install -e ./cocotb
+    sudo apt-get install make gcc g++ swig python3 python3-dev python3-pip
 
-.. note::
+Native Installation for Red Hat-based Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    After installation, you should be able to execute ``cocotb-config``.
-    If it is not found, you need to append its location to the ``PATH`` environment variable.
-    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented :ref:`here <python:inst-alt-install-user>`.
+.. code-block:: bash
 
-Native Linux Installation
--------------------------
+    sudo yum install make gcc gcc-c++ libstdc++-devel swig python3 python3-devel python3-pip
 
-The following instructions will allow building of the cocotb libraries
-for use with a 64-bit native simulator.
+macOS Packages
+^^^^^^^^^^^^^^
 
-If a 32-bit simulator is being used then additional steps are needed, please see
-`our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
+You need a few packages installed to get cocotb running on macOS.
+Installing a package manager really helps things out here.
 
-Debian/Ubuntu-based
-^^^^^^^^^^^^^^^^^^^
+`Brew <https://brew.sh/>`_ seems to be the most popular, so we'll assume you have that installed.
 
 .. code-block:: bash
 
-    sudo apt-get install git make gcc g++ swig python-dev
+    brew install python icarus-verilog gtkwave
 
-Red Hat-based
-^^^^^^^^^^^^^
+.. seealso::
 
-.. code-block:: bash
+   For more installation options (also for 32-bit simulators) please see `our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
 
-    sudo yum install gcc gcc-c++ libstdc++-devel swig python-devel
 
+Simulator Installation
+^^^^^^^^^^^^^^^^^^^^^^
 
-Windows Installation
---------------------
+For detailed instructions and the list of supported simulators see :ref:`simulator-support`.
 
-Download the MinGW installer from https://osdn.net/projects/mingw/releases/.
-
-Run the GUI installer and specify a directory you would like the environment
-installed in. The installer will retrieve a list of possible packages, when this
-is done press "Continue". The MinGW Installation Manager is then launched.
+.. note::
 
-The following packages need selecting by checking the tick box and selecting
-"Mark for installation"
+    Ensure that path to the simulator executable is appended to the ``PATH`` environment variable.
 
-.. code-block:: bash
 
-    Basic Installation
-      -- mingw-developer-tools
-      -- mingw32-base
-      -- mingw32-gcc-g++
-      -- msys-base
```

Comment:
Are we sure we want to remove these instructions? Maybe worth putting them on the wiki first, so they don't get lost completely.

Reply:
I think yes. On the wiki, I can add msys2 instructions.

Reply:
Added instruction for MSYS/MINGW here: https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions#windwows-installation-using-msys2